### PR TITLE
across(fix): change my deposit to position size

### PR DESCRIPTION
**Pull request by Gamaranto**
Signed-off-by: Gamaranto <francesco@umaproject.org>

### CODE REVIEW

File: src/components/PoolForm/PoolForm.tsx [hunk 1]
```diff
@@ -158,27 +158,21 @@ const PoolForm: FC<Props> = ({
       </Info>
       <Position>
         <PositionItem>
-          <div>My deposit</div>
+          <div>Position Size</div>
           <div>
-            {/* This can never be < 0, but for some weird iOS bug it shows -0 on mobile */}
-            {formatUnits(position, decimals).replace("-", "")} {symbol}
+            {formatUnits(position.add(feesEarned), decimals).replace("-", "")}{" "}
```
Comment (daywiss) [inline src/components/PoolForm/PoolForm.tsx:163]: just want to flag this, there is a value coming from the sdk already called `positionValue`. This is the variable we want in here, adding position and fees will produce a wrong value in some cases.

**Review by daywiss — CHANGES_REQUESTED** [review]
will need to update the var in position size

File: src/components/PoolForm/PoolForm.tsx (hunk 1 above)
Comment (daywiss) [reply src/components/PoolForm/PoolForm.tsx:163]: https://github.com/across-protocol/across-frontend/blob/master/src/views/Pool.tsx#L124

Comment (Gamaranto) [reply src/components/PoolForm/PoolForm.tsx]: Thanks, fixed

**Review by daywiss — APPROVED** [review]

**Pull request merged**